network/definition/data/templates/messages.yaml

```diff
@@ -61,3 +61,18 @@ messages:
         template: nvmRecordFailedInit
       nvmRecordsVersionFailed:
         template: nvmRecordsVersionFailed
+  wheelSpeed:
+    cycleTimeMs: 50
+    signals:
+      wheelSpeedLinearL:
+        template: wheelSpeedLinear
+      wheelSpeedLinearR:
+        template: wheelSpeedLinear
+      wheelSpeedRotationalL:
+        template: wheelSpeedRotational
+      wheelSpeedRotationalR:
+        template: wheelSpeedRotational
```

Comment:
I'm really not convinced you'll need anything other than rpm

Reply:
What do you ever need linear velocity for? The only thing I can think of is traction control. If you only need it in one place, calculate it in that place. RPM is much more useful for pretty much everything else, I think.